objects/src/accounts/testing/builders/mod.rs

```diff
@@ -1,6 +1,9 @@
 use core::fmt::Display;
 
-use miden_objects::{crypto::merkle::MerkleError, AccountError, AssetVaultError};
+use crate::{crypto::merkle::MerkleError, AccountError, AssetVaultError};
+
+pub mod account;
+pub mod account_id;
```

Comment:
In order to flatten the export structure a bit, is it possible to instead do:

```rust
pub use account::AccountBuilder;
pub use account_id::AccountIdBuilder;
```

Reply:
Yeah, I was planning that for the refactors (next PR). Essentially this PR is just moving code around for now.